wiki/Basic_MP3_Modifications/en.md

```diff
@@ -17,36 +19,39 @@ First, this guide assumes a few things:
 
 -   The MP3 has a known static BPM
 -   You have a high quality source
--   You want to cut the <u>middle</u> of the song, not the beginning, or the end
+-   You want to cut the middle of the song, not the beginning, or the end
 
 If at all possible you should use **[mp3DirectCut](http://www.mpesch3.de1.cc/mp3dc.html)**, which allows lossless editing. However when cutting a song, it's *not* lossless, as the data stream is not at the same rate as the BPM of the music. You can fudge along and approximate, but then it's *wrong*. I'd rather have a slight quality loss over a BPM that changes from 0 ms to +5 ms after the cut.
```

Comment:
>I'd rather have a slight quality loss over a BPM that changes from 0 ms to +5 ms after the cut.

This can be changed to remove the first person narrative